src/events.js

```diff
@@ -72,26 +70,29 @@ function handleMoveEvents(state, event, options) {
   state.hovered = elements;
 
   const context = {state, event};
-  dispatchMoveEvents(context, 'leave', previous, elements);
-  dispatchMoveEvents(context, 'enter', elements, previous);
+  return dispatchMoveEvents(context, 'leave', previous, elements) || dispatchMoveEvents(context, 'enter', elements, previous);
 }
 
 function dispatchMoveEvents({state, event}, hook, elements, checkElements) {
+  let changed;
   for (const element of elements) {
     if (checkElements.indexOf(element) < 0) {
-      dispatchEvent(element.options[hook] || state.listeners[hook], element, event);
+      changed = changed || dispatchEvent(element.options[hook] || state.listeners[hook], element, event);
```

Comment:
here too, dispatchEvent will not be called anymore when changed is true.

Reply:
should be ok  now